platform/core/src/utils/addUserPreferences.js

```diff
@@ -0,0 +1,7 @@
+import redux from './../redux';
+
+const addUserPreferences = ({ store, preferences }) => {
+  store.dispatch(redux.actions.setUserPreferences(preferences));
```

Comment:
This currently allows _any_ object for preferences. Should we deconstruct and make sure we're only setting for the general and window level top level keys so more state doesn't sneak in?

Reply:
Is `addUserPreferences` an exposed ActionCreator? Should we expose it in a way that makes that more obvious? (Utils feels like a weird place for this)

Reply:
I've remove it.